ui/src/hooks/useFileManage.ts

```diff
@@ -343,6 +345,7 @@ const initSocketEvent = (socket: WebSocket, t: any) => {
         fileManageStore.setCurrentPath('');
 
         socket.close();
+        lunaCommunicator.sendLuna(LUNA_MESSAGE_TYPE.FILE_MANAGE_EXPIRED, '');
         break;
       }
 
```

Comment:
The provided code appears to be incomplete, possibly due to it being written as part of a text-based communication rather than generated programmatically. I cannot perform in-depth checks on the code unless you include relevant code blocks here. If this is related to Vue.js, Node.js, or any other programming languages/ frameworks, could you please add more context?